apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/plugin/loader/AgentClassLoader.java

```diff
@@ -92,15 +92,14 @@ public static AgentClassLoader getDefault() {
      * @return
```

Comment:
Remove return if you want to change it to void.

Reply:
done!